packages/core/src/routes/applications/application.ts

```diff
@@ -127,12 +128,17 @@ export default function applicationRoutes<T extends AuthedRouter>(
       status: [200, 422],
     }),
     async (ctx, next) => {
-      const { oidcClientMetadata, ...rest } = ctx.guard.body;
+      const { oidcClientMetadata, protectedAppMetadata, ...rest } = ctx.guard.body;
 
       if (!EnvSet.values.isDevFeaturesEnabled && rest.type === ApplicationType.Protected) {
         throw new Error('dev feature for protected app is not enabled');
       }
 
+      assertThat(
```

Comment:
should also guard the patch method?

Reply:
Updated, this field is removed from patch guard until the patch route is ready